docs/examples/general/expressions/expr_examples.ipynb

```diff
@@ -4,26 +4,32 @@
    "cell_type": "markdown",
```

Comment:
<p><code><code></code>Values of Python's&nbsp;<code>int</code>&nbsp;and&nbsp;<code>float</code>&nbsp;types that are used directly;V</code><code></code></p><p> </p><p>trailing ;V?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2480/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Done</p>